.travis.yml

```diff
@@ -2,28 +2,44 @@ language: c
 compiler: gcc
 matrix:
   include:
-    - script: scripts/test-codingstyle.sh
-    - script: scripts/test-codingstyle-py.sh
-      cache: pip
-      install:
-        - pip install --user pylint
-        - pip install --user pycodestyle
-        - pip install --user jsonschema
-    - script: scripts/test-build.sh
-      cache: ccache
-      addons:
-        apt:
-          packages:
-            - libjson-c-dev
-            - uuid-dev
-            - doxygen
-            - python-jsonschema
-    - script: scripts/test-gtapi-mock-drv.sh
-      cache: ccache
-      addons:
-        apt:
-          packages:
-            - libjson-c-dev
-            - uuid-dev
-            - doxygen
-            - python-jsonschema
+  - script: scripts/test-codingstyle.sh
+  - script: scripts/test-codingstyle-py.sh
```

Comment:
Is there a difference to grouping the `script` items under `include:` vs. a level higher? Not sure what `include:` does.

After this change, do we still nee the `include:` line (4)?

Reply:
LInes 5 is one matrix/job configuration, line 6 to 11 is another.
The real change for the matrix configurations is the addition of the documentation job (lines 32-42).
I added to `include` key originally to explicitly group job configurations (instead of mixing/combining different matrix job settings).

Reply:
@luebbers I tested it manually by running with tagged commits as well as untagged commits in order to verify whether the proper conditions are checked.